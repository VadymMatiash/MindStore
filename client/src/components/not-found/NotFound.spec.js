import React from 'react'
import { shallow } from 'enzyme'
import NotFound from './NotFound'

describe('NotFound', () => {
  const notFound = shallow(<NotFound />)

  it('renders properly', () => {
    expect(notFound).toMatchSnapshot()
  })

  it('contains a not found header', () => {
    expect(notFound.find('h1').text()).toEqual('Page Not Found')
  })

  it('contains info text', () => {
    expect(notFound.find('p').text()).toEqual('Sorry, this page does not exist')
  })
})