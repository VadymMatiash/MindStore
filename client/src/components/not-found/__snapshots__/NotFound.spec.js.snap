// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders properly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1
          className="display-4"
>
          Page Not Found
</h1>,
        <p>
          Sorry, this page does not exist
</p>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Page Not Found",
          "className": "display-4",
        },
        "ref": null,
        "rendered": "Page Not Found",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Sorry, this page does not exist",
        },
        "ref": null,
        "rendered": "Sorry, this page does not exist",
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1
            className="display-4"
>
            Page Not Found
</h1>,
          <p>
            Sorry, this page does not exist
</p>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Page Not Found",
            "className": "display-4",
          },
          "ref": null,
          "rendered": "Page Not Found",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Sorry, this page does not exist",
          },
          "ref": null,
          "rendered": "Sorry, this page does not exist",
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
